wh11/js/main.js

```diff
@@ -0,0 +1,95 @@
+// Дан масив [16,-37,54,-4,72,-56,47,4, -16,25,-37,46,4,-51,27,-63,4,-54,76,-4,12,-35,4,47]
+// Знайти суму та кількість позитивних елементів.
+let arr = [16,-37,54,-4,72,-56,47,4, -16,25,-37,46,4,-51,27,-63,4,-54,76,-4,12,-35,4,47];
+let arr1 = arr.filter((el) => {
+    return el > 0;
+});
+let positiveLength = arr1.length;
+console.log('Кількість позитивних елементів = ',positiveLength);
+let sum = arr1.reduce((result,el) => {
+    return result += el;
```

Comment:
не треба змінювати аргументи фунції - це погана практика, хоча і цілком валідний код
```suggestion
    return result + el;
```